pastas/stressmodels.py

```diff
@@ -880,6 +889,52 @@ def get_parameters(self, model=None, istress=None):
             p_with_r = np.r_[p, distances]
         return p_with_r
 
+    def variance_gain(self, ml, istress=None):
```

Comment:
Can you remove this method here? I think it would be better to do something in ml.fit.get_ci_step_response(), or make a new function there, get_ci_gain or something, but apart from this PR. Would be good to think about a more general implementation first. Maybe add this code to the notebook?

Reply:
I'll remove it here and add it to the notebook for now. I'll open a separate issue for uncertainty and (Hantush)Wellmodel.